src/page/product-list.page.ts

```diff
@@ -1,14 +1,25 @@
-import { $, ElementFinder, ExpectedConditions, browser } from 'protractor';
+import { $$, ExpectedConditions, browser, ElementArrayFinder } from 'protractor';
 
 export class ProductListPage {
-  private addToCart: ElementFinder;
 
-  constructor () {
-    this.addToCart = $('a.button.ajax_add_to_cart_button.btn.btn-default');
+  private products: ElementArrayFinder;
+
+  constructor() {
+    this.products = $$('.product-container');
```

Comment:
can you exlpain me what the `$$` does? is this a shortcut for something?

Reply:
according to documentation, it replaces element.all(by.css()) (so it's still a css selector, only now it returns an array instead of a single element)